### PR TITLE
removing dev_agreement column

### LINKED ISSUE #8649: write an sql migration to drop `dev_agreement` from addons table
follow up from #8640 to land in a different tag/deploy.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #8649

### CODE REVIEW

**Comment by diox** [comment]
It's a column in the `addons` table, not a table.

**Comment by championshuttler** [comment]
@diox is it fine now?

File: src/olympia/migrations/1026-remove-dev-agreement.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `addon`
```
Comment (diox) [inline src/olympia/migrations/1026-remove-dev-agreement.sql]: The table is called `addons` with an `s` at the end.

File: src/olympia/migrations/1026-remove-dev-agreement.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+ DROP COLUMN `dev_agreement`
```
Comment (diox) [inline src/olympia/migrations/1026-remove-dev-agreement.sql]: The statement needs to end with a `;`

**Comment by championshuttler** [comment]
@diox Done :)

**Pull request merged**